front/src/containers/Notification/index.js

```diff
@@ -0,0 +1,182 @@
+// React imports
+import React from 'react'
+import { ScrollView } from 'react-native'
+
+// third imports
+import IconOcticons from 'react-native-vector-icons/Octicons'
+
+//local imports
+import {
+  GlobalStyle,
+
+  Container,
+  ContainerList,
+  Text,
+  Avatar,
+  Reminder
+} from './style'
+import user from '../../static/images/person.png'
+
+const NotificationScreen = () => {
+  return (
+    <Container>
+      <ScrollView>
+        <ContainerList onPress={() => alert('clicou')}>
```

Comment:
Rename it to something like ListItem, because when you use ContainerList, looks that is a full list, but it's only a item of a list

Reply:
Remove onPress with alert